lib/ami_spec/server_spec.rb

```diff
@@ -28,7 +28,11 @@ def run
       end
 
       $LOAD_PATH.unshift(@spec) unless $LOAD_PATH.include?(@spec)
-      require File.join(@spec, 'spec_helper')
+      begin
+        require File.join(@spec, 'spec_helper')
+      rescue LoadError
+        puts 'Spec Helper does not exist. Skipping!'
+      end
```

Comment:
Hmm I didn't catch this at first glance. I've questioned if we really need to do this, since the specs should load the spec helper.